src/CoreWCF.WebHttp/src/CoreWCF/OpenApi/OpenApiOptions.cs

```diff
@@ -70,5 +71,10 @@ public sealed class OpenApiOptions
         /// Any tags to hide.
         /// </summary>
         public IEnumerable<string> TagsToHide { get; set; } = new List<string>();
+
+        /// <summary>
+        /// Optional callback to sort tags.
+        /// </summary>
+        public Func<IEnumerable<OpenApiTag>, IEnumerable<OpenApiTag>> SortTags { get; set; }
```

Comment:
Did you consider having a property of type `IComparable<OpenApiTag>` so that a comparer can be provided? That way you can instantiate a `List<OpenApiTag>` and once they've all been populated, you can call `List<OpenApi>.Sort(IComparer<OpenApi> comparer)` if a comparer has been provided.  
I can think of a couple of reasons you might not want to do that.  If the required order might be non-trivial to represent in an IComparer (although I can't think of a scenario where I couldn't write it without a small amount of code). You might want to have this be a back door mechanism to filter the OpenApiTag instances or to enable extra ones being added. But that seems like a bad idea to me as you are encouraging breaking someone who depends on that if the implementation changes.